Description: Ignore the public api test
 It will install a whole rust toolchain using the rustup library
Author: NoisyCoil <noisycoil@tutanota.com>
Forwarded: no
Last-Update: 2025-01-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -85,10 +85,6 @@
 name = "error_handling"
 path = "tests/error_handling.rs"
 
-[[test]]
-name = "public_api"
-path = "tests/public_api.rs"
-
 [dependencies.bincode]
 version = "1.0"
 optional = true
@@ -147,21 +143,12 @@
 [dev-dependencies.pretty_assertions]
 version = "0.6"
 
-[dev-dependencies.public-api]
-version = "0.33.1"
-
 [dev-dependencies.rayon]
 version = "1.0.0"
 
 [dev-dependencies.regex]
 version = "1.0"
 
-[dev-dependencies.rustdoc-json]
-version = "0.8.8"
-
-[dev-dependencies.rustup-toolchain]
-version = "0.1.5"
-
 [features]
 default = ["default-fancy"]
 default-fancy = [
